packages/shared/src/hooks/useFeedSettings.ts

```diff
@@ -64,7 +69,7 @@ export default function useFeedSettings(): FeedSettingsReturnType {
 
       return { ...req, feedSettings };
     },
-    { ...disabledRefetch },
+    { ...disabledRefetch, enabled },
```

Comment:
Should we consider adding staleTime here as well? 
My only note on that field would be to validate if we manually reset/mutate the settings on change.

Reply:
Yeah, I was thinking of that too. Maybe we should start considering it on most of our queries, especially the tooltip-related query.

Would 60 seconds be sufficient for this one?

Reply:
We should indeed, it can also be part of the global research for the loads, so no need to do for all.
The time totally depends on the type of query, this one needs to "behave" realtime, meaning if people make changes it should reflect.
So the main thing is we need to validate that happens (any tag changes etc should reflect invaldiation/mutation) if that happens we can go with 60 (I think I set a default on another query already, which we can reuse)

Reply:
Awesome. I will check if invalidation happens whenever we send the relevant mutations.

Reply:
After checking, the `useMutateFilters` manually updates the local cache so no need for invalidation.

advanced settings:
https://github.com/dailydotdev/apps/blob/dfa97065f51a017db5babf2f8bda03570c6f5dda/packages/shared/src/hooks/useMutateFilters.ts#L85-L99

tags:
https://github.com/dailydotdev/apps/blob/dfa97065f51a017db5babf2f8bda03570c6f5dda/packages/shared/src/hooks/useMutateFilters.ts#L106-L120

sources:
https://github.com/dailydotdev/apps/blob/dfa97065f51a017db5babf2f8bda03570c6f5dda/packages/shared/src/hooks/useMutateFilters.ts#L127-L141